ci/clang-formater/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM debian:unstable-20211011-slim
```

Comment:
why this one?

Reply:
Didn't want the waste time on it xD.
On the stable one the clang is not available on apt repo.